cloud/gcp/modules/compute/variables.tf

```diff
@@ -0,0 +1,44 @@
+variable "region" {
+  type        = string
+  description = "Default region for the application"
+}
+
+variable "db_connection_name" {
+  type        = string
+  description = "DB connection string"
+}
+
+variable "bucket_name" {
+  type        = string
+  description = "Name of the file storage bucket"
+}
+
+variable "http_port" {
+  type        = string
+  description = "Port to open on container for HTTP requests"
+}
+
+variable "application_service_account_email" {
+  type        = string
+  description = "The service account application runs with"
+}
+
+variable "db_secret_id" {
```

Comment:
can this be read from the secret manager directly? if absolutely not possible at least mark it as sensitive?

Reply:
got it, that makes sense!